source/write/run-command.txt

```diff
@@ -0,0 +1,185 @@
+``Cursor`` object.
+
+The following code shows how you can use the ``command()``
+method on a database to run the ``hello`` command, which returns
+information about the current member's role in the replica set:
+
+.. code-block:: php
+
+   $result = $database->command(['hello' => 1]);
+
+To find a link to a full list of database commands and corresponding
+parameters, see the :ref:`Additional Information section
+<php-addtl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   The ``command()`` method does not
+   obey the read preference you might have set on your ``Database``
+   instance elsewhere in your code. By default, ``command()`` uses the ``primary``
+   read preference.
+   
+   You can set a read preference for command execution by setting one
+   in the options parameter, as shown in the following code:
+   
+   .. code-block:: php
+
+      $readPref = new MongoDB\Driver\ReadPreference('primaryPreferred');
+      $result = $database->command(
+          ['ping' => 1], 
+          ['readPreference' => $readPref]
+      );
+
+   For more information on read preference options, see :manual:`Read
+   Preference </core/read-preference/>` in the {+mdb-server+} manual.
+
+.. _php-command-response:
+
+Response
+--------
+
+The ``command()`` method returns a ``Cursor`` object that contains
+the response from the database after the command has been executed.
+
+Each database command performs a different function, so the response
+content can vary depending on the command executed. However, every
+response contains a document with the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field
+     - Description
+
+   * - <command result>
+     - Fields specific to the database command. For example,
+       ``count`` returns the ``n`` field and ``explain`` returns the
+       ``queryPlanner`` field.
+
+   * - ``ok``
+     - Whether the command has succeeded (``1``)
+       or failed (``0``).
+
+   * - ``operationTime``
+     - The logical time of the operation. MongoDB uses the
+       logical time to order operations. To learn more about this
+       concept, see our blog  post about the :website:`Global Logical
+       Clock </blog/post/transactions-background-part-4-the-global-logical-clock>`.
+
+   * - ``$clusterTime``
+     - A document that contains the signed cluster time. Cluster time is a
+       logical time used for the ordering of operations.
+
+       This document contains the following fields:
+
+       - ``clusterTime``, the timestamp of the highest known cluster time for the member
+       - ``signature``, a document that contains the hash of the cluster time and the ID
+         of the key used to sign the cluster time
+
+.. _php-command-example:
+
+Command Example
+---------------
+
+The following example uses the ``command()`` method to run
+the ``explain`` command for a ``count`` operation. The ``explain``
+command runs in the ``queryPlanner`` verbosity mode:
```

Comment:
Note that the method `Collection::explain` exists.

Reply:
I see. I can use a different example, such as `dbStats` or something similar?